examples/k8s/daemonset/daemonset.yaml

```diff
@@ -57,11 +57,11 @@ metadata:
   namespace: default
 spec:
   selector:
-    matchLabels:
+    matchAttributes:
       name: stanza
   template:
     metadata:
-      labels:
+      attributes:
```

Comment:
I don't believe either of these should change, as they are k8s keys.

Reply:
@jsirianni Can you confirm?

Reply:
Correct, avoid modifying any Kubernetes configs in. `examples/` 👍